backend/src/controllers/user.ts

```diff
@@ -210,6 +211,60 @@ const deactivateUserCallback: RequestHandler = async (req, res, next) => {
   }
 }
 
+const addFavQues: RequestHandler = async (req, res, next) => {
+  let resp: ReturnResponse;
+  const userId = req.userId;
+  const options = req.body.options;
+  const question = req.body.question;
+
+  try {
+    const user = await User.findById(userId);
+    if (!user) {
+      const err = new ProjectError("User does not exist");
+      err.statusCode = 401;
+      throw err;
+    }
+    
+    const favQues = new Fav({ question, options,userId });    const favQuestion = await favQues.save();
+    resp = { status: "success", message: "Question added to Favourites!", data: {} };
+    res.status(200).send(resp);
+  }
+  catch (error) {
+    next(error);
+  }
+};
+
+const showFavQues: RequestHandler = async (req, res, next) => {
+  const userId = req.userId;
+  let resp: ReturnResponse;
+  try {
+    const fav = await Fav.find({userId}); 
```

Comment:
this should be named favQuestion, follow a meaningful naming habit.

Reply:
noted